engines/order_management/spec/services/order_management/subscriptions/summary_spec.rb

```diff
@@ -81,6 +90,17 @@ module Subscriptions
           summary.instance_variable_set(:@success_ids, success_ids)
           expect(summary.issue_count).to be 2 # 7 & 9
         end
+
+        context "when there are also subscription issues" do
+          let(:subscription_issues) { [101, 212] }
+
+          it "includes subscription issues in the count" do
+            summary.instance_variable_set(:@order_ids, order_ids)
+            summary.instance_variable_set(:@success_ids, success_ids)
+            summary.instance_variable_set(:@subscription_issues, subscription_issues)
```

Comment:
It's pretty dangerous to mess with someone else's encapsulation. What if we inject we call `#record_order` `#record_success` and `#record_subscription_issue` in the test set up instead?